docs/UserGuide.md

```diff
@@ -64,6 +64,21 @@ If you are an **experienced user**:
 
 1. Refer to the [Features](#features) below for details of each command.
 
+--------------------------------------------------------------------------------------------------------------------
+
+## Prefix Summary
+| Prefix | Attribute   | Constraint                                                   |
+|--------|-------------|--------------------------------------------------------------|
+| `\d`   | Department  | Alphanumeric characters and whitespaces only                 |
```

Comment:
should we keep it consistent and use an alphabet followed by the forward slash instead? e.g. "n/" and "i/"